internal/core/testdata/test-template-printer-template-list-with-options.golden

```diff
@@ -0,0 +1,15 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟩🟩🟩 STDOUT️ 🟩🟩🟩️
+David Copperfield <-> 111111111-111111111
```

Comment:
Now I kinda want to see what was the command from the golden file 🤔

Reply:
In this case: `scw list -o template={{ .Name }} <-> {{ .ID }}`
If you want to open a ticket to include in the golden the command feel free to do so :)